homeassistant/components/enocean/__init__.py

```diff
@@ -10,28 +13,46 @@
 from .const import DATA_ENOCEAN, DOMAIN, ENOCEAN_DONGLE
 from .dongle import EnOceanDongle
 
+_LOGGER = logging.getLogger(__name__)
+
 CONFIG_SCHEMA = vol.Schema(
     {DOMAIN: vol.Schema({vol.Required(CONF_DEVICE): cv.string})}, extra=vol.ALLOW_EXTRA
 )
 
 
+# component setup
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the EnOcean component."""
+
+    if not hass.data.get(DOMAIN):
+        services.async_setup_services(hass)
+
     # support for text-based configuration (legacy)
     if DOMAIN not in config:
+        # no config entry for our domain available, e.g. in
```

Comment:
These comments are not needed. This code is pretty standard for integrations. Code comments are needed for things where the integration does something out of the ordinary.